grafana/dashboards/daemonset.json

```diff
@@ -2133,7 +2133,7 @@
         "multi": false,
         "name": "namespace",
         "options": [],
-        "query": "label_values(request_total{daemonset!=\"\"}, namespace)",
+        "query": "label_values(tcp_open_total{daemonset!=\"\"}, namespace)",
```

Comment:
Per in-person convo, it would be great to use `process_start_time_seconds` for these queries, since that will populate with all instances that have reported metrics to prometheus, rather than just those that have made network connections.